### ISSUE TITLE
No Speech feedback for lines with start or end on another voice

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28933]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

If a line such as 8va, diminuendo, etc starts and ends on a beat in voice 2 that also has a note on that beat in voice 1, then both voices when you arrow along them correctly report the start and end of the line by the screen reader.

If the line starts or ends in voice 2 on a beat that is mid way through a note in voice 1 then if you arrow along voice 1 you will not hear where the start or end is.

STEPS.
(See the attached example)
1.  Arrow along voice 1 and then voice 2.  Notice that in bar 1 and 2 you can correctly hear where the 8va and the hairpin begin and end on both voices.  Because both voices have a note in that position.
2. Arrow along each voice in bar 3 and you will see that voice 1 has no indication of any sort that there is a diminuendo effecting it.  Voice 2 reads correctly.
3. Arrow along both voices in bar 4 and notice that the end of the 8va is reported in Voice 1, but you have no idea where it has started.

In this last example if the 8va was extended into the next bar and ended on a mid note position of voice 1 then the user would not know that these notes are not written at the octave they are stated at.

Suggestions of what the speech could say in the voice with the mid note are:
*  At the end of all other info attached to the note immediately before the diminuendo (example bar 3) "diminuendo hairpin starts mid way".
* or more accurately, still attached to note before, "hairpin diminuendo starts beat 2.5".

### Supporting files, videos and screenshots

[LinesAndVoices.zip](https://github.com/user-attachments/files/21422233/LinesAndVoices.zip)

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
This seems similar to the behaviour in #28037 and #19244 where elements in a mid-note position can't be read by the screen reader.  It would seem sensible to use the same output for this once determined.
